app/ui.R

```diff
@@ -35,6 +35,11 @@ ui <- dashboardPage(
       menuItem(
         "Background", 
         tabName = "background", 
+        icon = icon("book-open")
+      ),
+      menuItem(
```

Comment:
I don't think you need a second tab with references, all this information is in the "background" section. Did anyone specifically ask for this?